apps/ios/Shared/MediaEncryption.playground/Contents.swift

```diff
@@ -0,0 +1,78 @@
+import UIKit
+import SimpleXChat
+
+hs_init(0, nil)
+
+let totalBytes = 20
+let ivTagBytes = 28
+var base: UnsafeMutableRawPointer = malloc(totalBytes)
+
+let assume = base.assumingMemoryBound(to: UInt8.self)
+assume[0] = 0 // key frame
+for i in 1..<totalBytes {
+    assume[i] = UInt8(i)
+}
+let unencrypted = NSData(bytesNoCopy: base, length: totalBytes)
+let aesKey = "PI-bV-FTgRqZM_lsDH9T21a0yRVMsvLFmvilJ9Ssk3g="
```

Comment:
Just wondering if it's okay to expose this.

Reply:
This is just a playground to test the code, it’s not used in the app.